cfme/infrastructure/openstack_node.py

```diff
@@ -32,3 +38,31 @@ def run_introspection(self):
         view.toolbar.configuration.item_select('Introspect Node', handle_alert=True)
         exp_msg = '"{}": Introspect successfully initiated'.format(self.name)
         view.flash.assert_success_message(exp_msg)
+
+
+@attr.s
+class OpenstackNodeCollection(HostCollection):
+    """Collection object for the :py:class:`cfme.infrastructure.openstack_node.OpenstackNode`."""
+
+    ENTITY = OpenstackNode
+
+
+@navigator.register(OpenstackNodeCollection)
```

Comment:
This is duplicating the 'All' destination registered against HostCollection, and shouldn't be necessary.

https://github.com/ManageIQ/integration_tests/blob/master/cfme/infrastructure/host.py#L571